cloudlift/deployment/service_creator.py

```diff
@@ -165,3 +168,13 @@ def _print_progress(self):
             log_err("Finished with status: %s" % (final_status))
         else:
             log_bold("Finished with status: %s" % (final_status))
+
+    def service_update_preflight_checks(self):
```

Comment:
Move pre-flight checks to config/preflight.py file

Reply:
done